common/changes/@bentley/measure-tools-react/measure-tools_2021-05-17-20-42.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@bentley/measure-tools-react",
+      "comment": "Initial release of bentley/measure-tools-react!",
+      "type": "minor"
```

Comment:
@calebmshafer i was hoping to go with major release, 1.0.0 but when i did rush change i didnt see major as an option. im guessing its cause if the locked major in the version policy. Any recommendations? publish this to make sure it works before 1.0.0 or?

Reply:
I'd push out a 0.x version and then do the major version bump.

When you do go to bump it, the version policy lock will need to be updated.